features/openhab-addons/src/main/feature/feature.xml

```diff
@@ -220,6 +220,11 @@
         <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.pulseaudio/${project.version}</bundle>
     </feature>
 
+    <feature name="openhab-binding-regoheatpump" description="RegoHeatPump Binding" version="${project.version}">
+        <feature>openhab-runtime-base</feature>
```

Comment:
> I need to specify it here in this case too?

Well, you do not have any way to tell, which "case" you are in. You need to declare the dependencies that your binding needs to properly function. And if it can happen that the user defines a serial connection, he will run into errors, if this dependency is not present.